src/funcx_common/task_storage/s3.py

```diff
@@ -1,5 +1,7 @@
 import typing as t
 
+import botocore.exceptions
```

Comment:
This should be in the `try` imports, right?
(Realistically, the first users of this stuff will always be using the `boto3` extra, but I think it still pays to move this.)

Reply:
Good catch. Fixed now.